books/free-programming-books-es.md

```diff
@@ -327,6 +327,7 @@
 ### Python
 
 * [Aprenda a pensar como un programador (con Python)](https://argentinaenpython.com/quiero-aprender-python/aprenda-a-pensar-como-un-programador-con-python.pdf) (PDF)
+* [Aprende Python](https://aprendepython.es) (HTML)
```

Comment:
Add author attributions to fit with licenses

```suggestion
* [Aprende Python](https://aprendepython.es) - Sergio Delgado Quintero (HTML)
```

Reply:
Done! Thank you!